src/containers/AppBody.jsx

```diff
@@ -112,8 +112,8 @@ export class AppBody extends Component {
 
         const permissions = {
           canRead: true,
-          // Currently, users may only edit a file they own
-          canEdit: (username === cookie.load('username')),
+          canEdit: (username === cookie.load('username') ||
```

Comment:
This is kind of confusing with username meaning 2 different things.  Should it be tokenUser, tokenOrgs, and snippetUser?